domain.md

```diff
@@ -0,0 +1,71 @@
+
+# 1) Creating an account with delegated permissions to join a computer to domain. 
+
+The following steps setup an account with delegated privileges to join a computer to domain, assuming an account already created, otherwise, please create an account first.
+
+ * Login Domain Controller Server.
+ * Lanuch Active Directory Users and Computers.
+ * Locate the **OU** that you want to join a computer to and right-click and then click Delegate Control. 
+ * The **Delegation of Control Wizard** opens, click **Next**
+ * In the **Users or Groups** page, click **Add** to add a specific user or a specific group to the **Selected users and groups** list, and then click **Next**
+ * In the **Tasks to Delegate** page, click **Create a custom task to delegate**, and then click **Next**.
+ * In the **Active Directory Object Type** page, Click **Only the following objects in the folder**, and then from the list, click to select the **Computer objects** check box. Then, select the check boxes below the list, **Create selected objects in this folder** and **Delete selected objects in this folder**, Click **Next**.
+ * In the **Permissions** page, click to select the following permission check boxes from list:  
+       &nbsp;&nbsp;&nbsp;&nbsp;**Reset Password**  
+       &nbsp;&nbsp;&nbsp;&nbsp;**Read and write Account Restrictions**  
+       &nbsp;&nbsp;&nbsp;&nbsp;**Validated write to DNS host name**  
+       &nbsp;&nbsp;&nbsp;&nbsp;**Validated write to service principal name**
+ * Click **Next** to **the Completing the Delegation of Control Wizard** page, after review then click **Finish**.
+
+
+# 2) Enable remote powershell admin on a Domain Controller (DC) machine. 
+
+The following steps enable remote powershell admin
+
+* Login Domain Controller Server.
+* Start Windows PowerShell as an administrator.
+* Check the configuration of WinRM service by running following command:  
+  **Get-WmiObject Win32_Service | Where-Object { $_.Name -eq 'WinRM'} | Format-Table**  
+  The value of the StartMode property in the output should be "Auto" and the value of the Status property in the output should be "Running"
+* Set the StartMode property of WinRM service to Automatic by running following command if needed:  
+  **Set-Service WinRM -StartupType Automatic**  
+  **Restart-Service -Name WinRM**
+* run command to enable remote powershell: **Enable-PSRemoting -Force**
+
+
+# 3) Enable LDAPs on a on Domain Controller (DC) machine. 
+    
+Basically, there are two methods of enabling LDAPS on a DC.
+
+* Method #1: install an Enterprise Root CA  
+  The first method is the easiest: LDAPS is automatically enabled when you install an Enterprise Root CA on a Domain Controller. If you install the AD-CS role and specify the type of setup as ��Enterprise�� on a DC, all DCs in the forest will be automatically be configured to accept LDAPS.  
+    
+  
+* Method #2: add a Digital Certificate on DC  
+  Requirements for an LDAPS certificate:  
```

Comment:
Is this from somewhere or did you assemble this from a bunch of snippets? If  there's a good MSFT/other doc that has most of the same info, would be good to reference it as well.

Reply:
I searched a lot online, basically it looks the same. Create a CA with specially requirement. and install to the machine, giving the permission to access private key.